packages/office-ui-fabric-react/src/components/TextField/docs/TextFieldBestPractices.md

```diff
@@ -0,0 +1,11 @@
+### Layout
+
+- Use a multiline text field when long entries are expected.
+- Don't place a text field in the middle of a sentence, because the sentence structure might not make sense in all languages. For example, "Remind me in [textfield] weeks" should instead read, "Remind me in this many weeks [textfield]".
```

Comment:
```suggestion
- Don't place a text field in the middle of a sentence, because the sentence structure might not make sense in all languages. For example, "Remind me in [textfield] weeks" should instead read, "Remind me in this many weeks: [textfield]".
```

Reply:
Nice! Fixed.